services/src/handlers/ebv.rs

```diff
@@ -201,6 +209,44 @@ async fn get_ebv_dataset<C: Context>(
     Ok(web::Json(dataset))
 }
 
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+struct EbvHierarchy {
+    provider_id: DatasetProviderId,
+    tree: NetCdfOverview,
+}
+
+async fn get_ebv_subdatasets<C: Context>(
+    dataset_name: web::Path<String>,
+    _params: web::Query<()>,
+    _base_url: web::Data<BaseUrl>,
+    _session: C::Session,
+    _ctx: web::Data<C>,
+) -> Result<impl Responder> {
+    dbg!(&dataset_name);
+
+    let listing = {
+        // TODO: make dir configurable
+        let data_path = test_data!("netcdf4d/").join(dataset_name.into_inner());
+
+        debug!("Accessing dataset {}", data_path.display());
+
+        crate::util::spawn_blocking(move || NetCdfCfDataProvider::build_netcdf_tree(&data_path))
+            .await
+            // TODO: error handling
+            .unwrap()
+            .unwrap()
+    };
+
+    // TODO: find a way to get the external dataset provider id
+    let provider_id = DatasetProviderId::from_str("1690c483-b17f-4d98-95c8-00a64849cd0b")?;
```

Comment:
maybe we can hardcode it (here or in the dataset provider). I guess there will ever be only one ebv provider so the id doesn't have to be part of the provider definition json

Reply:
6d935f0